locales/en-US/server.ftl

```diff
@@ -0,0 +1,205 @@
+shotPageShareEmail
+  .title = Share link via email
+shotPageShareLink = Get a shareable link to this shot:
+shotPagePrivacyMessage = Anyone with the link can view this shot.
+shotPageCopyImageText
+  .label = Copy image text
+shotPageConfirmDeletion = Are you sure you want to delete this shot permanently?
+// Note: {$timediff} is a placeholder for a future relative time clause like 'in 3 days' or 'tomorrow'
+shotPageExpirationMessage = If you do nothing, this shot will be permanently deleted {$timediff}.
+// Note: {$date} is a placeholder for a localized future date as returned by Date.toLocaleString.
+// For example, in en-US, {$date} could be "7/12/2017, 1:52:50 PM".
+shotPageRestoreButton = restore until {$date}
+shotPageExpiredMessage = This shot has expired.
+// Note: This phrase is followed by an empty line, then the URL of the source page
+shotPageExpiredMessageDetails = Here is the page it was originally created from:
+shotPageDeleteButton
+  .title = Delete this shot
+shotPageAbuseButton
+  .title = Report this shot for abuse, spam, or other problems
+shotPageDownloadShot
+  .title = Download
+shotPageDownload = Download
+// Note: do not translate 'Firefox Screenshots' when translating this string
+shotPageScreenshotsDescription  = Screenshots made simple. Take, save, and share  screenshots without leaving Firefox.
+shotPageUpsellFirefox = Get Firefox now
+shotPageDoesNotExpire = Does not expire
+shotPageDMCAMessage = This shot is no longer available due to a third party intellectual property claim.
+// Note: {$dmca} is a placeholder for a link to send email (a 'mailto' link)
+shotPageDMCAContact = Please email {$dmca} to request further information.
+// Note: do not translate 'Firefox Screenshots' when translating this string
+shotPageDMCAWarning = If your Shots are subject to multiple claims, we may revoke your access to Firefox Screenshots.
+// Note: {$url} is a placeholder for a shot page URL
+shotPageDMCAIncludeLink = Please include the URL of this shot in your email: {$url}
+shotPageKeepFor = How long should this shot be retained?
+// Note: shotPageSelectTime is a placeholder label for the time selection dropdown.
+shotPageSelectTime = Select time
+shotPageKeepIndefinitely = Indefinitely
+shotPageKeepTenMinutes = 10 Minutes
+shotPageKeepOneHour = 1 Hour
+shotPageKeepOneDay = 1 Day
+shotPageKeepOneWeek = 1 Week
+shotPageKeepTwoWeeks = 2 Weeks
+shotPageKeepOneMonth = 1 Month
+shotPageSaveExpiration = save
+shotPageCancelExpiration = cancel
+shotPageDoesNotExpire = does not expire
+// Note: {$timediff} is a placeholder for a future relative time clause, like "in 1 week" or "tomorrow"
+shotPageExpiresIn = expires {$timediff}
+// Note: {$timediff} is a placeholder for a past relative time clause, like "1 week ago" or "yesterday"
+shotPageExpired = expired {$timediff}
+timeDiffJustNow = just now
+// TODO: should we just have one string with plural rules?
+timeDiffOneMinuteAgo = 1 minute ago
+timeDiffMinutesAgo = {$number} minutes ago
```

Comment:
This is likely related to indentation in the FTL file. We should have better error reporting for this. The entire body of the translation which follows the `=` must be indented in Fluent:

This is correct:

```
timeDiffMinutesAgo = { $number ->
        [one] 1 minute ago
       *[other] {$number} minutes ago
    }
```

This is correct as well (and my preferred style):

```
timeDiffMinutesAgo =
    { $number ->
        [one] 1 minute ago
       *[other] {$number} minutes ago
    }
```

This OTOH results in a Syntax error because the closing `}` is not indented.

```
timeDiffMinutesAgo = { $number ->
    [one] 1 minute ago
   *[other] {$number} minutes ago
}
```

There's an open syntax spec issue about this https://github.com/projectfluent/fluent/issues/42 although I'm not sure we should allow this. It creates an inconsistency and I prefer to only allow it once we've depleted other options for preventing this error (error reporting, linting, prittifying).